lib/vsc/filesystem/gpfs.py

```diff
@@ -244,7 +246,7 @@ def _executeY(self, name, opts=None, prefix=False):
             opts = list(opts)
         else:
             self.log.error("_executeY: have to use a list or tuple for options: name %s opts %s" % (name, opts))
-            return
+            return None
```

Comment:
`return {}` would be better. Unless you want to crash the caller.

Reply:
Which is what is done now as well ...

Reply:
the new tests in vsc-install require explicit returns on the whole function if you start returning

Reply:
Yes, so `return {}`